config/processEimage.py

```diff
@@ -0,0 +1 @@
+config.calibrate.photoCal.doSelectUnresolved=False
```

Comment:
How about this, I'll assign a code review to someone else and build a new stack in the mean time.  Then when I merge, hopefully it wouldn't be too painful for you to do the sanity check that this code works with the data we are using.